internal/flags/csvseparator.go

```diff
@@ -0,0 +1,33 @@
+package flags
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var csvSeparator rune
+var csvSeparatorValue string
```

Comment:
Why are both needed? It seems that csvSeparator is not really used. You could just remove it and instead in line 22 return rune(csvSeparatorValue[0]), nil